daprdocs/content/en/developing-applications/integrations/Azure/authenticating-azure.md

```diff
@@ -110,8 +110,8 @@ APP_NAME="dapr-application"
 # Create the app
 APP_ID=$(az ad app create \
   --display-name "${APP_NAME}" \
-  --available-to-other-tenants false \
-  --oauth2-allow-implicit-flow false \
+  --enable-access-token-issuance false \
+  --enable-id-token-issuance false
```

Comment:
These two flags seem pretty different and I don't think we should use them. I'd just remove the flags if they've been deleted.

Reply:
Both the `--enable....` parameters look like they would replace `--oauth2-allow-implicit-flow`, per [`az ad app`](https://docs.microsoft.com/en-us/cli/azure/ad/app?view=azure-cli-latest#az-ad-app-create-optional-parameters) reference... but I can remove if they aren't necessary for the example. What do you think?

Reply:
I think they're not needed actually, it should be ok without those flags.